rdagent/scenarios/data_science/proposal/exp_gen/proposal.py

```diff
@@ -1056,6 +1067,9 @@ def gen(
         else:
             inject_diverse = False
 
+        # add available packages prompt
+        packages_prompt = get_available_packages_prompt()
```

Comment:
```suggestion
        # TODO: check the availability of the packagess.  Otherwise, it will not match other environment.
        packages_prompt = get_available_packages_prompt()
```